src/IO/ClassIO/ImageIO.h

```diff
@@ -33,14 +33,14 @@ namespace three {
 
 /// The general entrance for reading an Image from a file
 /// The function calls read functions based on the extension name of filename.
-/// \return If the read function is successful.
+/// \return True if the read function is successful, false otherwise.
 bool ReadImage(const std::string &filename, Image &image);
 
 /// The general entrance for writing an Image to a file
 /// The function calls write functions based on the extension name of filename.
 /// If the write function supports quality, the parameter will be used.
 /// Otherwise it will be ignored.
-/// \return If the write function is successful.
+/// \return True if the write function is successful, false otherwise.
```

Comment:
`return true if the write function is successful, false otherwise. `. The same thing applies for the other edits.

Reply:
fixed.